apps/readest-app/src/services/nativeAppService.ts

```diff
@@ -59,8 +59,9 @@ export const nativeFileSystem: FileSystem = {
     return URL.createObjectURL(new Blob([content]));
   },
   async copyFile(srcPath: string, dstPath: string, base: BaseDir) {
-    const { fp, baseDir } = resolvePath(dstPath, base);
-    await copyFile(srcPath, fp, base && { toPathBaseDir: baseDir });
```

Comment:
As I read from the documentation copyFile will avoid file data copying from rust to JavaScript then to rust again. It will just copy the file in rust land, isn't it?